tests/QPSolverTest.cpp

```diff
@@ -634,6 +634,94 @@ BOOST_AUTO_TEST_CASE(QPDamperJointLimitsTest)
 	BOOST_CHECK_EQUAL(solver.nrConstraints(), 0);
 }
 
+BOOST_AUTO_TEST_CASE(QPMimicJointTest)
+{
+	using namespace Eigen;
+	using namespace sva;
+	using namespace rbd;
+	using namespace tasks;
+	namespace cst = boost::math::constants;
+
+	/* Make a very simple gripper */
+	MultiBodyGraph mbg;
+
+	double mass = 1.;
+	Matrix3d I = Matrix3d::Identity();
+	Vector3d h = Vector3d::Zero();
+
+	RBInertiad rbi(mass, h, I);
+
+	Body b0(rbi, "b0");
+	Body b1(rbi, "b1");
+	Body b2(rbi, "b2");
+
+	mbg.addBody(b0);
+	mbg.addBody(b1);
+	mbg.addBody(b2);
+
+	Joint j0(Joint::RevZ, true, "j0");
+	Joint j1(Joint::RevZ, true, "j1");
+	j1.makeMimic("j0", -1.0, 0.0);
+
+	mbg.addJoint(j0);
+	mbg.addJoint(j1);
+
+	PTransformd from = sva::PTransformd::Identity();
+
+	mbg.linkBodies("b0", from, "b1", from, "j0");
+	mbg.linkBodies("b1", from, "b2", from, "j1");
+
+	MultiBody mb = mbg.makeMultiBody("b0", true, from);
+
+	MultiBodyConfig mbcInit(mb);
+	mbcInit.zero(mb);
+
+	forwardKinematics(mb, mbcInit);
+	forwardVelocity(mb, mbcInit);
+
+	std::vector<rbd::MultiBody> mbs = {mb};
+	std::vector<rbd::MultiBodyConfig> mbcs = {mbcInit};
+
+	qp::QPSolver solver;
+
+	tasks::qp::PostureTask pt(mbs, 0, mbcInit.q, 100.0, 10.0);
+
+	solver.nrVars(mbs, {}, {});
+	solver.updateConstrSize();
+
+	solver.addTask(&pt);
+	BOOST_CHECK_EQUAL(solver.nrTasks(), 1);
+
+
+	// Test mimic joint in mbc
+	mbcs[0] = mbcInit;
+	for(int i = 0; i < 1000; ++i)
+	{
+		BOOST_REQUIRE(solver.solve(mbs, mbcs));
+		eulerIntegration(mbs[0], mbcs[0], 0.001);
+
+		forwardKinematics(mbs[0], mbcs[0]);
+		forwardVelocity(mbs[0], mbcs[0]);
+		BOOST_REQUIRE(mbcs[0].q[1][0] == -mbcs[0].q[2][0]);
+	}
+	BOOST_CHECK_SMALL(mbcs[0].q[1][0], 1e-4);
+
+	mbcs[0] = mbcInit;
+	pt.posture({{}, {0.2}, {-0.2}});
+	for(int i = 0; i < 1000; ++i)
+	{
+		BOOST_REQUIRE(solver.solve(mbs, mbcs));
+		eulerIntegration(mbs[0], mbcs[0], 0.001);
+
+		forwardKinematics(mbs[0], mbcs[0]);
+		forwardVelocity(mbs[0], mbcs[0]);
+		BOOST_REQUIRE(mbcs[0].q[1][0] == -mbcs[0].q[2][0]);
+	}
+	BOOST_CHECK_SMALL(mbcs[0].q[1][0] - 0.2, 1e-4);
+
+	solver.removeTask(&pt);
+	BOOST_CHECK_EQUAL(solver.nrTasks(), 0);
+}
```

Comment:
I remember we had a few problems when alpha = 0, could it be possible to add one such joint to the test?